pkg/cmd/render/render.go

```diff
@@ -9,7 +9,9 @@ import (
 	"encoding/pem"
 	"errors"
 	"fmt"
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/configobservation/auth"
```

Comment:
I don't maintain this repository - @tkashem to decide whether he wants 3rd party libraries mixed among system libs in imports